_projects/2018/timelab/quasiperiodic.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Phase-resolved oscillations
+desc: Implement method to calculate the phase of oscillatory phenomena with non-constant frequency, and calculate phase-resolved spectra.
+requirements:
+ - Familiar with numerical analysis
+difficulty: medium
+issues:
+ - https://github.com/StingraySoftware/stingray/issues/293
+mentors:
+ - matteobachetti
```

Comment:
Will also @abigailStev mentoring this? we prefer 2 mentor per projects if possible (one as a backup)

Reply:
She will actually be the main mentor, I think. I just put my name as a placeholder :)